ExtraDry/ExtraDry.Blazor/Components/Dry/DryStaticFilter.razor.cs

```diff
@@ -0,0 +1,42 @@
+namespace ExtraDry.Blazor;
+public partial class DryStaticFilter<TItem> : ComponentBase, IExtraDryComponent
+{
+    public DryStaticFilter()
+    {
+        ViewModelDescription = new ViewModelDescription(typeof(TItem), this);
+    }
+
+    /// <inheritdoc cref="IExtraDryComponent.CssClass "/>
+    [Parameter]
+    public string CssClass { get; set; } = string.Empty;
+
+    /// <inheritdoc cref="IExtraDryComponent.UnmatchedAttributes" />
+    [Parameter(CaptureUnmatchedValues = true)]
+    public Dictionary<string, object>? UnmatchedAttributes { get; set; }
+
+    [Parameter]
+    public int? InitialHierarchyLevel { get; set; }
+
+    [Parameter]
+    public string? TextFilter { get; set; }
+
+    [Parameter]
+    public Dictionary<string,object>? PropertyFilter { get; set; }
```

Comment:
I'm not even sure what the right interface for this should be.  Let's not make it a contract until we get there, and remove it.

Reply:
Removed